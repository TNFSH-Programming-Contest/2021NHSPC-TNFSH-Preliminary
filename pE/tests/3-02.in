501 1000
3 4 16
5 6 18
6 7 19
8 9 20
8 9 28
10 11 11
3 4 18
12 13 18
3 8 22
9 4 26
6 7 22
14 15 9
14 15 15
14 16 19
17 18 6
19 20 28
12 13 21
21 18 20
12 13 29
14 21 29
14 15 20
12 13 8
22 5 26
14 17 30
4 23 13
24 25 14
26 7 22
4 23 17
14 15 8
14 16 12
15 16 7
27 28 17
24 25 14
14 16 26
23 29 10
3 4 8
18 16 12
30 27 28
31 32 19
33 34 14
14 16 16
3 29 14
25 35 18
36 37 15
13 19 16
12 13 19
36 37 11
22 26 14
3 29 27
38 39 19
36 37 28
16 40 14
36 37 25
41 42 7
3 29 18
43 44 11
35 30 20
35 30 29
22 7 9
16 38 23
28 45 10
39 40 16
46 12 20
3 47 30
12 13 15
42 24 28
45 31 11
36 37 5
14 16 7
22 26 19
48 49 7
47 29 30
50 46 24
51 52 8
34 48 10
22 53 17
37 50 6
40 36 22
52 54 16
55 56 6
10 11 10
57 33 8
58 59 12
14 36 29
32 60 23
35 32 12
49 61 26
41 35 13
52 54 5
52 54 17
10 11 10
62 57 22
63 14 18
64 3 13
61 65 14
43 44 13
10 11 13
10 11 25
62 14 11
41 60 20
65 14 19
22 63 10
36 20 22
11 41 17
44 66 12
53 67 10
66 54 16
14 20 18
7 67 8
51 54 20
22 62 9
68 51 19
22 62 7
64 29 26
56 69 9
22 51 24
22 68 7
70 71 19
72 73 10
22 14 17
69 10 15
70 74 28
75 43 9
76 77 22
60 72 17
22 51 7
71 74 15
22 78 19
22 75 12
22 14 16
67 20 5
77 64 10
22 79 26
22 55 27
22 79 24
74 80 6
22 60 29
73 81 25
22 20 28
22 54 10
80 58 27
59 60 16
82 83 9
29 54 5
78 84 21
20 70 19
85 82 9
83 60 30
79 60 22
76 54 16
84 60 24
81 76 20
22 60 14
22 60 16
22 85 16
81 76 22
22 60 22
60 81 17
86 54 25
22 86 12
87 22 17
87 54 17
87 88 19
87 88 22
54 88 25
87 88 19
87 89 19
90 91 11
92 93 8
87 88 9
89 88 24
89 88 5
87 88 8
87 88 23
92 93 12
87 88 20
91 87 10
88 94 27
90 91 16
92 93 16
91 95 16
87 92 27
93 94 26
87 94 24
96 97 12
87 96 6
98 87 28
95 98 30
97 94 27
87 94 24
90 91 23
99 100 7
94 101 16
87 102 10
102 101 7
101 100 8
99 90 6
99 100 16
103 104 5
105 106 13
105 106 26
107 105 23
108 107 22
109 103 16
108 105 9
110 108 22
111 99 16
106 112 5
111 99 16
99 100 6
100 110 20
112 113 5
113 114 18
115 116 26
109 104 10
117 109 17
116 114 29
115 111 28
118 117 6
114 118 8
119 115 17
118 104 16
118 104 22
119 104 6
104 120 21
121 122 20
120 123 7
119 120 19
120 123 5
122 120 30
121 120 22
121 124 6
124 125 27
126 121 16
125 119 22
121 120 9
121 120 26
121 120 7
120 123 12
127 126 19
120 123 17
120 123 21
123 128 22
120 123 13
121 120 5
121 120 20
127 128 13
127 128 19
127 128 6
127 128 25
127 128 6
127 129 12
128 129 24
130 127 20
131 130 28
131 129 20
129 132 19
131 129 23
132 133 11
132 133 21
131 132 19
131 133 9
131 133 19
131 133 15
131 133 19
131 134 12
134 133 25
131 133 5
135 133 17
131 135 5
136 137 21
133 136 7
137 138 15
131 138 27
139 140 30
141 142 26
131 138 23
140 131 19
138 141 12
143 139 30
142 144 8
145 144 20
145 144 17
145 143 20
145 146 18
146 144 24
147 144 24
145 147 30
148 145 11
148 145 6
148 149 11
149 144 11
150 151 18
150 151 16
150 152 11
152 153 13
153 148 6
148 144 10
150 152 18
151 144 14
152 144 12
150 154 21
154 144 5
144 155 23
150 144 17
150 144 28
156 155 7
144 155 25
144 155 21
144 157 28
150 144 16
158 159 12
157 156 13
144 157 19
150 155 27
159 150 30
158 155 24
160 155 20
158 160 19
158 155 30
161 158 23
162 163 14
164 162 16
164 162 10
164 162 18
165 166 14
167 168 13
169 170 8
165 171 17
172 169 9
163 172 23
170 173 10
170 173 13
161 155 23
164 173 20
165 171 13
165 166 8
171 166 13
173 168 28
164 173 18
165 166 17
174 161 21
167 168 14
173 168 6
165 175 20
175 166 9
174 161 27
176 177 15
176 164 20
155 165 22
164 173 23
178 179 26
176 180 26
181 182 28
165 166 11
176 180 9
174 161 30
183 184 18
184 168 20
176 168 22
167 168 23
174 161 7
176 167 19
176 181 18
179 183 27
182 168 18
166 177 8
174 161 6
176 182 6
178 168 12
166 177 26
176 178 7
161 166 8
185 174 7
176 180 23
186 187 5
180 186 6
180 186 28
168 177 26
187 185 10
168 177 28
176 177 25
188 189 15
190 189 11
190 189 20
176 177 10
191 176 30
190 188 7
192 190 10
193 192 25
192 189 13
194 191 19
193 192 28
193 192 22
195 189 28
177 193 14
194 195 12
196 194 10
189 197 21
196 197 11
196 189 13
198 199 30
200 196 22
197 199 23
200 198 17
201 202 26
199 203 20
200 199 8
202 204 11
205 206 26
205 206 19
207 204 23
208 201 7
200 203 9
209 200 29
210 211 21
206 209 22
212 213 15
204 200 14
212 210 19
212 210 8
202 207 9
202 204 8
204 205 21
206 209 13
204 200 8
203 212 9
214 215 12
208 202 24
211 214 22
213 210 18
208 202 10
200 215 23
215 216 18
217 218 11
216 219 18
208 200 10
220 221 27
221 222 25
208 219 23
220 221 20
223 208 26
224 217 7
225 226 10
225 226 18
220 221 9
220 221 26
227 228 20
220 221 8
220 221 10
222 225 25
225 226 8
220 221 9
229 230 14
231 232 26
223 219 11
233 226 12
234 218 14
229 235 24
228 229 11
229 235 23
229 235 11
227 228 10
227 228 15
233 226 21
230 235 14
219 236 6
237 224 13
220 233 18
229 235 28
228 238 18
235 238 15
220 226 28
234 218 28
239 223 23
234 218 8
240 227 29
220 240 7
241 239 30
242 243 22
243 234 14
231 232 17
242 237 15
244 245 24
220 226 22
232 246 8
242 218 15
220 226 18
232 246 5
236 231 26
247 242 17
232 246 24
246 248 12
249 244 10
249 245 16
245 247 26
238 226 9
218 241 26
226 249 8
218 248 11
248 250 14
220 250 5
220 250 5
220 250 18
251 252 8
253 252 28
251 252 24
254 253 25
255 252 6
254 255 23
220 250 12
256 257 20
258 259 20
258 260 29
261 256 16
261 256 19
260 251 21
257 262 23
259 254 24
263 264 20
250 258 25
265 263 27
261 256 6
261 256 6
266 264 20
267 220 9
268 266 15
261 265 27
269 267 23
261 256 10
270 271 6
262 272 21
273 269 8
252 274 13
261 268 15
270 271 14
270 271 20
275 270 17
271 276 17
277 272 20
278 273 29
279 280 11
275 271 24
281 282 22
278 274 29
264 277 14
281 283 23
278 274 6
278 274 26
282 283 18
272 279 29
276 274 28
284 278 27
280 284 16
279 280 24
285 272 26
279 280 6
281 261 24
278 275 27
281 272 20
283 285 10
281 272 24
279 278 20
278 274 11
281 274 17
281 274 5
286 274 5
281 279 6
287 288 13
289 287 19
287 288 30
281 286 30
281 274 15
274 289 19
289 288 13
281 290 7
290 288 7
290 288 23
281 288 15
291 281 7
292 291 20
288 293 6
294 293 15
292 293 15
294 292 16
294 293 26
295 296 23
293 297 26
294 298 5
299 298 17
294 299 8
300 295 13
298 297 9
300 296 30
294 297 17
296 301 24
301 294 9
300 294 17
298 297 7
294 297 8
294 297 20
294 297 9
300 297 24
302 300 26
297 303 30
302 303 6
304 305 14
302 303 22
302 303 11
304 302 17
305 303 8
304 303 17
304 303 25
303 306 9
306 307 27
304 307 24
304 307 14
304 307 15
308 304 10
307 309 25
308 309 14
308 309 22
308 309 5
310 311 14
308 309 24
312 310 5
308 309 26
309 310 13
309 312 22
309 311 16
311 313 16
313 314 14
308 314 16
313 314 14
308 314 14
315 316 25
308 314 5
316 308 9
317 318 11
315 308 7
318 319 15
308 314 11
318 319 12
319 315 20
314 320 6
317 320 28
321 322 18
323 321 19
324 317 6
325 323 11
323 322 29
324 325 28
322 320 8
326 324 17
327 326 28
327 328 26
329 327 7
330 331 6
328 320 29
332 329 19
320 330 20
333 332 11
333 331 22
333 331 9
333 331 15
334 333 17
333 331 25
335 334 9
335 331 23
334 331 13
335 331 29
336 337 28
338 339 21
340 338 24
337 340 15
331 336 24
339 341 14
342 335 19
341 343 30
342 335 13
342 343 23
343 344 5
345 346 5
342 347 24
345 346 6
345 346 10
344 347 26
347 348 7
349 345 18
350 351 12
352 353 30
351 353 20
350 353 19
350 352 13
349 348 5
349 348 21
346 348 6
354 350 14
349 342 18
355 356 22
357 358 19
355 356 9
349 353 29
349 353 23
359 358 22
348 354 9
360 358 26
354 353 27
357 360 27
356 358 6
361 358 19
357 358 18
361 358 30
356 358 24
359 358 30
354 353 26
357 361 17
362 363 9
364 358 26
357 364 21
355 358 19
359 358 8
365 366 13
367 349 16
368 366 20
368 365 10
369 370 20
370 371 29
372 367 15
373 374 20
357 364 14
367 353 13
362 363 17
375 373 20
376 366 19
363 372 12
369 371 20
359 358 22
359 358 24
377 355 6
378 379 20
380 369 9
359 358 23
357 358 8
381 382 19
359 358 7
371 376 27
368 366 22
357 362 8
371 366 23
353 359 30
383 384 27
380 368 29
357 358 21
382 383 10
385 374 16
381 382 5
386 385 28
387 388 6
375 389 14
357 358 20
375 386 20
375 374 12
375 374 23
390 391 14
381 382 6
390 391 21
392 393 28
366 378 25
379 394 23
395 396 8
377 358 27
391 397 9
392 393 17
389 398 24
398 374 12
375 374 8
399 381 27
388 400 30
395 401 14
395 401 16
357 402 5
396 401 22
379 394 9
403 404 14
397 405 26
397 405 17
393 406 23
402 377 5
357 358 27
379 394 28
407 408 14
409 390 28
410 384 30
411 387 19
392 406 25
399 410 28
412 411 12
413 414 22
415 416 30
366 379 17
417 380 10
413 414 23
407 408 22
418 419 25
420 392 7
400 421 18
375 374 23
408 418 25
399 384 23
417 394 28
417 394 26
401 420 22
415 416 19
401 420 13
375 374 8
375 374 9
422 423 14
424 425 30
426 427 18
358 401 13
427 422 19
401 420 23
409 405 17
358 395 27
428 419 6
423 357 18
413 414 17
405 417 30
424 425 9
423 357 17
417 394 29
429 2 13
408 419 6
424 425 11
404 430 16
394 431 22
424 425 29
430 412 7
424 432 5
384 426 16
399 384 11
433 429 20
423 357 28
429 2 14
406 413 26
429 2 24
407 419 11
434 435 7
415 416 6
424 432 26
375 436 28
437 438 25
425 2 15
414 424 6
407 428 19
436 374 10
439 440 14
419 433 15
375 374 14
375 374 22
441 442 11
443 415 22
374 403 25
424 440 21
432 425 26
444 375 6
438 445 27
446 447 10
443 416 18
448 449 29
450 447 21
440 2 12
419 2 23
443 416 20
442 451 30
424 439 5
443 416 14
452 407 15
431 453 10
454 455 24
435 456 24
424 2 6
446 454 24
450 447 18
457 441 29
458 459 27
460 461 8
462 437 30
451 463 20
455 464 20
465 444 14
466 424 5
466 399 30
448 467 22
424 462 27
445 452 29
449 467 25
461 468 29
421 469 16
470 471 23
409 431 12
446 450 12
472 448 6
464 473 26
458 459 11
446 447 7
457 451 29
446 447 11
466 424 28
458 459 12
448 467 9
470 471 10
453 474 26
469 475 14
457 451 28
443 416 18
434 456 29
476 416 11
443 476 6
460 472 13
474 463 5
477 448 13
473 447 6
471 477 28
443 478 9
479 443 14
458 459 5
480 481 23
482 483 20
451 409 11
475 434 17
443 416 6
467 457 19
478 484 15
431 474 10
447 485 22
486 487 11
443 416 11
460 448 10
479 443 7
460 448 5
483 488 5
481 463 28
1 458 6
465 475 5
489 480 27
465 475 25
485 486 23
443 416 21
490 470 25
459 447 20
475 2 7
468 490 11
491 2 10
456 2 24
487 482 16
447 487 11
477 448 30
451 463 7
492 465 11
493 479 16
484 416 11
466 424 8
465 2 7
494 495 30
496 497 11
460 489 5
465 2 14
495 492 18
460 489 18
416 466 23
482 497 18
443 416 5
458 459 22
460 463 18
465 498 14
1 458 20
494 499 30
499 465 25
500 460 18
488 496 12
458 459 7
443 416 15
446 447 26
443 416 12
501 458 19
465 2 28
458 447 15
463 2 22
416 491 13
465 500 5
497 494 24
1 501 20
465 2 17
1 446 20
447 482 11
458 447 23
498 493 20
443 416 6
