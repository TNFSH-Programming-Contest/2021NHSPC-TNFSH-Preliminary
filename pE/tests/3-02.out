82.6479106497402131529
